mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java

```diff
@@ -151,6 +153,29 @@ private void initiateCenterCreation(CenterPayload centerPayload) {
 
         if (isCenterNameValid()) {
             mCreateNewCenterPresenter.createCenter(centerPayload);
+            AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(getActivity(),
+                    R.style.MaterialAlertDialogStyle);
+            alertDialogBuilder
+                    .setMessage(getString(R.string.create_center_dialog))
+                    .setPositiveButton(getString(R.string.action_yes),
+                            new DialogInterface.OnClickListener() {
+                                @Override
+                                public void onClick(DialogInterface dialogInterface,
+                                                    int i) {
+                                    et_centerName.setText("");
+                                    cb_centerActiveStatus.setChecked(false);
```

Comment:
Rename these to `etCenterName`, `cbCenterActiveStatus` and similarly, `btnSomeButton` `tvSomeTextView` etc for the other views in this file.